src/goal.cpp

```diff
@@ -127,6 +132,30 @@ CommandCost CmdRemoveGoal(DoCommandFlag flags, GoalID goal)
 	return CommandCost();
 }
 
+/**
+ * Update goal destination of a goal.
+ * @param flags type of operation
+ * @param goal GoalID to update.
+ * @param type GoalType of destination.
+ * @param dest GoalTypeID of destination.
+ * @return the cost of this operation or an error
+ */
+CommandCost CmdSetGoalDestination(DoCommandFlag flags, GoalID goal, GoalType type, GoalTypeID dest)
+{
+	if (_current_company != OWNER_DEITY) return CMD_ERROR;
+	if (!Goal::IsValidID(goal)) return CMD_ERROR;
+	Goal *g = Goal::Get(goal);
+	if (!Goal::IsValidGoalDestination(g->company, type, dest)) return CMD_ERROR;
+
+	if (flags & DC_EXEC)
+	{
```

Comment:
Opening brackets should not be on their own line, except for the opening bracket of functions.

Reply:
Thanks, I've fixed it.